MainApp/MainApp/FeatureNetwork/ProfileNetwork/ProfileNetwork.swift

```diff
@@ -8,6 +8,68 @@
 
 import Foundation
 
-class ProfileNetwork {
+protocol ProfileLandingNetworkDelegate: class {
+    func didSuccessRetrieveProfile(response: GetProfileResponse)
+    func didFailedRetrieveProfile(error: Error)
+}
+
+protocol EditProfileNetworkDelegate: class {
+    func didSuccessEditProfile(response: EditProfileResponse)
+    func didFailedEditProfile(error: Error)
+}
+
+protocol ProfileLandingNetworkProtocol {
+    var profileLandingNetworkDelegate: ProfileLandingNetworkDelegate? { get set }
+    
+    func profileGet()
+}
+
+protocol EditProfileNetworkProtocol {
```

Comment:
combine profileLanding and editProfile to ProfileNetwork

Reply:
combined